app/api/collection.ts

```diff
@@ -70,9 +96,21 @@ class CollectionAPI extends PlutoAxios {
         throw new Error("Collection API's getPapers method is broken.");
       }
     });
-    const camelizedData = camelcaseKeys(resData, { deep: true });
-    const normalizedData = normalize(camelizedData, [paperInCollectionSchema]);
-    return normalizedData;
+    const camelizedPaperData = camelcaseKeys(resPaperData, { deep: true });
```

Comment:
I suggest using `camelcaseKeys` once. If you call `camelcaseKeys(res.data.data, { deep: true });` all child keys be camelized.